Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:slim
```

Comment:
Let's use a specific version of node e.g `FROM node:8.9.4-alpine`

Reply:
Can you explain why?

Reply:
Sure, There is no way to know what version of node we are using when `FROM node:slim` 
Would be nice to always install the version required/suggested and easy to know from the dockerfile

Reply:
Well, I don't have such a such a strong position there, I just like to keep these things up-to-date :) But I'll change it and we make sure to update it on a regular basis

Reply:
🙏